jobs/async-upload/tests/test_models.py

```diff
@@ -0,0 +1,113 @@
+"""
+Unit tests for models.py - specifically testing model validation logic.
+"""
+import pytest
+from pydantic import ValidationError
+
+from job.models import ModelConfig, UploadIntent
+
+
+class TestModelConfigValidateModelIds:
```

Comment:
This sort of thing could probably all be reduced to two tests by parametrizing, but this is self-contained and short enough that I would just leave it as is for now.